tests/Feature/Api/Place/PlaceCategoryTagsControllerTest.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Tests\Feature;
+
+use Hedonist\Entities\User\User;
+use Illuminate\Support\Facades\DB;
+use Tests\Feature\Api\ApiTestCase;
+use Hedonist\Entities\Place\PlaceCategory;
+use Hedonist\Entities\Place\PlaceCategoryTag;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+
+class PlaceCategoryTagsControllerTest extends ApiTestCase
+{
+    use RefreshDatabase;
+
+    private $user;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->user = factory(User::class)->create();
+        $this->actingWithToken($this->user);
+    }
+
+    public function testGetTags()
+    {
+        $category = PlaceCategory::create([
+            'name' => 'testCategory'
+        ]);
+
+        $tag = PlaceCategoryTag::create([
+            'name' => 'testCategoryTag'
+        ]);
+
+        DB::table('place_category_place_tag')->insert([
+            'place_category_id' => $category->id,
+            'place_tag_id'      => $tag->id
+        ]);
+
+        $this->assertDatabaseHas('place_category_place_tag',
+            [
+                'place_category_id' => $category->id,
+                'place_tag_id'      => $tag->id
+            ]
+        );
+
```

Comment:
empty line

Reply:
I leave it for more readable code structure)
Removed.